stock_packaging_calculator_packaging_level/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Camptocamp SA
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl)
+{
+    "name": "Stock packaging calculator packaging level",
+    "summary": "Glue module for packaging level",
+    "version": "18.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Warehouse Management",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "application": False,
+    "installable": True,
+    "auto_install": True,
```

Comment:
I would make it optional. It should be up to the user to choose to display by packaging name or level code
```suggestion
    "auto_install": False,
```